### PR TITLE
Feature/us5777 requestor sets event dates

### PR DESCRIPTION
**Pull request by JoeKer1**

### CODE REVIEW

File: crossroads.net/app/mp_tools/add_event_tool/add_event/add_event.html [hunk 1]
```diff
@@ -308,6 +309,6 @@
         </span>
       </ng-messages>
     </div>
-
+    <div>{{evt.eventData | json}}</div>
```
Comment (ilpadre) [inline crossroads.net/app/mp_tools/add_event_tool/add_event/add_event.html]: Did you mean to leave this in here?

File: crossroads.net/app/mp_tools/add_event_tool/add_event/addEvent.component.js [hunk 2]
```diff
@@ -1,215 +1,14 @@
...
+
+export default function addEventComponent() {
+  return {
+    restrict: 'E',
+    bindings: {
+      eventData: '='
+    },
+    template,
+    controller,
+    controllerAs: 'evt'
```
Comment (ilpadre) [inline crossroads.net/app/mp_tools/add_event_tool/add_event/addEvent.component.js]: Don't like abbreviations, especially for controller names.

File: crossroads.net/app/mp_tools/add_event_tool/add_event/add_event.html (hunk 1 above)
Comment (JoeKer1) [inline crossroads.net/app/mp_tools/add_event_tool/add_event/add_event.html]: not at all!

File: crossroads.net/app/mp_tools/add_event_tool/add_event/addEvent.component.js (hunk 2 above)
Comment (JoeKer1) [inline crossroads.net/app/mp_tools/add_event_tool/add_event/addEvent.component.js]: This is what it was before. I just left it that way so I didn't have to replace it in the template and miss one. I can change it.

**Pull request merged**